src/main/java/uk/gov/hmcts/reform/sscs/validators/SscsCaseValidator.java

```diff
@@ -241,23 +241,24 @@ private void checkRepresentative(Appeal appeal, Map<String, Object> ocrCaseData,
         }
     }
 
-    private void checkMrnDetails(Appeal appeal) {
+    private void checkMrnDetails(Appeal appeal, Map<String, Object> ocrCaseData) {
```

Comment:
This caseValidator is being call through bulkscan with ocrCaseData but it also being called to validate appeal where validate appeal doesn't have ocrCaseData. This logic will failed on validate appeal
Please put condition if the ocrCaseData doesn't exists using !doesIssuingOfficeExist(appeal) condition